app/Http/Controllers/TransactionController.php

```diff
@@ -81,9 +82,29 @@ public function show($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function edit($id)
+
+    private function getLastTransaction($account_id, $transaction_id){
```

Comment:
The name of the function is getLastTransaction, but it is returning a balance? I think we have to change the naming of that to getLastTransactionBalance?